classes/patterns/class-homepage.php

```diff
@@ -0,0 +1,151 @@
+											</div>
+											<div class="carousel-caption">
+												<div class="caption-overlay"></div>
+												<div class="container main-header">
+													<div class="carousel-captions-wrapper">
+														<h2></h2>
+														<p></p>
+													</div>
+												</div>
+											</div>
+										</div>
+									</div>
+								</div>
+							</div>
+						</section>
+					</div>
+				</div>
+				<!-- /wp:planet4-blocks/carousel-header -->
+
+				' . Issues::get_config()['content'] . '
+
+				<!-- wp:spacer {"height":"88px"} -->
+				<div style="height:88px" aria-hidden="true" class="wp-block-spacer"></div>
+				<!-- /wp:spacer -->
+
+				<!-- wp:planet4-blocks/articles {
+					"article_heading":"' . __( 'Read our Stories', 'planet4-blocks' ) . '"
+				} /-->
+
+				<!-- wp:spacer {"height":"56px"} -->
+				<div style="height:56px" aria-hidden="true" class="wp-block-spacer"></div>
+				<!-- /wp:spacer -->
+
+				' . SideImageWithTextAndCta::get_config()['content'] . '
+
+				<!-- wp:spacer {"height":"30px"} -->
+				<div style="height:30px" aria-hidden="true" class="wp-block-spacer"></div>
+				<!-- /wp:spacer -->
+
+				' . SideImageWithTextAndCta::get_config( [ 'media_position' => 'right' ] )['content'] . '
+
+				<!-- wp:spacer {"height":"56px"} -->
+				<div style="height:56px" aria-hidden="true" class="wp-block-spacer"></div>
+				<!-- /wp:spacer -->
+
+				<!-- wp:planet4-blocks/covers {"cover_type":"take-action","className":"is-style-take-action"} -->
+				<div class="wp-block-planet4-blocks-covers is-style-take-action" data-render="planet4-blocks/covers" data-attributes="{&quot;attributes&quot;:{&quot;cover_type&quot;:&quot;take-action&quot;,&quot;initialRowsLimit&quot;:1,&quot;title&quot;:&quot;&quot;,&quot;description&quot;:&quot;&quot;,&quot;tags&quot;:[],&quot;post_types&quot;:[],&quot;posts&quot;:[],&quot;version&quot;:2,&quot;layout&quot;:&quot;grid&quot;,&quot;isExample&quot;:false,&quot;readMoreText&quot;:&quot;Load more&quot;,&quot;className&quot;:&quot;is-style-take-action&quot;},&quot;innerBlocks&quot;:[]}"></div>
+				<!-- /wp:planet4-blocks/covers -->
+
+				<!-- wp:spacer {"height":"72px"} -->
+				<div style="height:72px" aria-hidden="true" class="wp-block-spacer"></div>
+				<!-- /wp:spacer -->
+
+				<!-- wp:group {"align":"full","backgroundColor":"grey-05"} -->
+				<div class="wp-block-group alignfull has-grey-05-background-color has-background">
+					<!-- wp:spacer {"height":"80px"} -->
+					<div style="height:80px" aria-hidden="true" class="wp-block-spacer"></div>
+					<!-- /wp:spacer -->
+
+					<!-- wp:group {"className":"container"} -->
+					<div class="wp-block-group container">
+
+					<!-- wp:columns -->
+					<div class="wp-block-columns">
+
+						<!-- wp:column -->
+						<div class="wp-block-column">
+
+							<!-- wp:heading {"level":2,"style":{"typography":{"fontSize":"40px"}},"placeholder":"' . __( 'Enter title', 'planet4-blocks-backend' ) . '"} -->
```

Comment:
I'm pretty sure that level 2 is the default one, so you can probably remove it from the attributes (not a big deal, but it simplifies the line a bit!)